env.example.yml

```
@@ -1,17 +1,5 @@
 LAMBDA_EXEC_SG: Insert AWS Security Group ID Here (it must be in the same VPC as the subnet)
 LAMBDA_EXEC_SUBNET: Insert AWS Subnet ID Here (it must be in the same VPC as the security group)
 BUGSNAG_NOTIFIER_KEY: INSERT BUGSNAG NOTIFIER KEY HERE
-GEOCODER_API_KEY: INSERT API KEY HERE
-GEOCODE_EARTH_URL: https://api.geocode.earth/v1 # Not needed if geocode.earth is not used
-CSV_ENABLED: true
-CUSTOM_PELIAS_URL: http://<insert your Pelias endpoint here>/v1
-GEOCODER: HERE # Options: HERE, PELIAS
-TRANSIT_GEOCODER: (optional) OTP/PELIAS
-TRANSIT_BASE_URL: (conditionally required) OTP instance when TRANSIT_GEOCODER=OTP (/otp/routers/{routerId})
-
-SECONDARY_GEOCODER: (optional) HERE/PELIAS
-SECONDARY_GEOCODER_API_KEY: (optional) INSERT SECONDARY API KEY HERE
-SECONDARY_GEOCODE_EARTH_URL: (optional) https://api.geocode.earth/v1 # Not needed if geocode.earth is not used
-
-REDIS_HOST: (optional) <insert IP of redis host here>
-REDIS_KEY: (optional) <insert redis password here>
+GEOCODERS: <JSON Array of OTP-UI `GeocoderConfig`s>
```

How can this be a JSON array?

---

It's stringified!

---

Okay. Can you just tweak:
```suggestion
GEOCODERS: <Stringified JSON Array of OTP-UI `GeocoderConfig`s>
```